quiche/src/h3/ffi.rs

```diff
@@ -227,7 +227,7 @@ pub extern "C" fn quiche_h3_send_request(
     let req_headers = headers_from_ptr(headers, headers_len);
 
     match conn.send_request(quic_conn, &req_headers, fin) {
-        Ok(v) => v as i64,
```

Comment:
TODO: if th FFI swallows this, then apps can't use it. Consider a way to expose it to apps